flyteadmin/pkg/errors/errors.go

```diff
@@ -181,7 +184,148 @@ func NewWorkflowExistsIdenticalStructureError(ctx context.Context, request *admi
 	return statusErr
 }
 
+func IsSameDCFormat(oldSpec *admin.LaunchPlanSpec, newSpec *admin.LaunchPlanSpec) bool {
+	oldParams := oldSpec.GetDefaultInputs().GetParameters()
+	newParams := newSpec.GetDefaultInputs().GetParameters()
+
```

Comment:
I am writing more comments here to let people understand how this algorithm works.